# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/accountsservice/fix-generate-version.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- accountsservice-23.13.9/generate-version.sh.vanilla	2024-06-23 10:30:52.096000000 +0200
+++ accountsservice-23.13.9/generate-version.sh	2024-06-23 10:31:20.062000000 +0200
@@ -8,7 +8,7 @@
 exec 2>&3
 
 # If it's not from a git checkout, assume it's from a tarball
-if [ "$TOPLEVEL_WORKING_DIR" != "$CWD" ]; then
+if [ ! -d ".git" ]; then
     VERSION_FROM_DIR_NAME=$(basename "$CWD" | sed -n 's/^accountsservice-\([^-]*\)$/\1/p')
 
     if [ -n "$VERSION_FROM_DIR_NAME" ]; then
